src/Accordion/Accordion.js

```diff
@@ -99,9 +100,15 @@ export default class Accordion extends Component {
 
   static Item = AccordionItem;
 
-  state = {
-    openList: [],
-  };
+  constructor(props) {
+    super(props);
+
+    const openByDefault = props.items.map((item, index) => (item.isOpenByDefault ? index : undefined));
+
+    this.state = {
+      openList: openByDefault.filter(e => e !== undefined),
```

Comment:
```suggestion
      openList: props.items.filter(item => item.isOpenByDefault),
```

Reply:
@choochootrain `#handleItemToggle` relies on an item's index to show/hide items so I don't think this suggestion would work.